crates/rover-client/src/shared/check_response.rs

```diff
@@ -312,10 +321,15 @@ impl ProposalsCheckResponse {
     }
 
     pub fn get_msg(&self) -> String {
-        match self.severity_level {
-            ProposalsCheckSeverityLevel::ERROR => "Your check failed because some or all of the diffs in this change are not in an approved Proposal.".to_string(),
-            ProposalsCheckSeverityLevel::WARN => "Your check passed with warnings because some or all of the diffs in this change are not in an approved Proposal.".to_string(),
-            ProposalsCheckSeverityLevel::OFF => "Proposal checks are disabled".to_string(),
+        match self.proposal_coverage {
+            ProposalsCoverage::FULL => "All of the diffs in this change are associated with an approved Proposal.".to_string(),
+            ProposalsCoverage::PARTIAL | ProposalsCoverage::NONE => match self.severity_level {
+                ProposalsCheckSeverityLevel::ERROR => "Your check failed because some or all of the diffs in this change are not in an approved Proposal.".to_string(),
+                ProposalsCheckSeverityLevel::WARN => "Your check passed with warnings because some or all of the diffs in this change are not in an approved Proposal.".to_string(),
```

Comment:
and "and your schema check severity level is set to WARN"

Reply:
Added in latest revision